packages/taucmdr/cf/software/papi_installation.py

```diff
@@ -70,6 +70,12 @@ def _prepare_src(self, *args, **kwargs):
         src_prefix = super()._prepare_src(*args, **kwargs)
         if os.path.basename(src_prefix) != 'src':
             src_prefix = os.path.join(src_prefix, 'src')
+        
+        # Modify PAPI's config.mk script to not use -Werror
```

Comment:
What warning is being thrown that is triggering the build error?

Reply:
There's a warning in PAPI only with NVHPC compilers. This is an upstream bug with PAPI.